src/makeRobotAccountant.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 1;
+  const errorMessage = 'Bzzz... Error!';
+
+  return (a) => {
+    return (b) => {
+      if (count === 4) {
+        count++;
+
+        return errorMessage;
+      } else if ((4 + (count % 2) > 4 || count < 4)) {
```

Comment:
simplify this line, it could be much simpler
if count <= 3 or count %2 !== 0

Reply:
I need more sleep.. 
Senks)